livebooks/markdown/stylesheets.md

```diff
@@ -464,7 +464,7 @@ Then use the `template` attribute on the view to be injected into the stylesheet
 
 ```heex
 <Image style="overlay-circle">
-  <Circle template="circle" style="stroke(.red, lineWidth: 4)" >
+  <Circle template="circle" style="stroke(.red, lineWidth: 4)"></Circle>
```

Comment:
This can just be self closing:

```heex
<Cirble template="circle" style="stroke(.red, lineWidth: 4)" />
```

Reply:
Oh, I see! It’s missing the backslash. By the way, how do you know when can be a self-closing element?

@bcardarella I just pushed a commit with the updated version.

Reply:
I believe any element can be self-closing if it doesn't necessitate any child content.

Reply:
@Francisco-Castro as @carson-katri said, I have been opting into self-closing when there are no children present as it just looks cleaner in the markup but you are free to use a closing tag if you want in your own apps!

Reply:
That makes sense. I like that approach of opting for self-closing tags when there are no more children present—it definitely makes the markup look cleaner. Thank you both for clarifying!